etc/error_code.toml

```diff
@@ -1,11 +1,11 @@
-["KV:Cloud:IO"]
+["KV:Cloud:Io"]
```

Comment:
why change

Reply:
https://github.com/tikv/tikv/pull/13215 changed all namings to use the camel case but without calling `make error-code`, which I did.

Reply:
That's intentional. Not sure if the error code is of case sensitivity. /cc @5kbpers

Reply:
It's OK because error codes are used for printing only currently.